Header header
float32 err_x
float32 err_y
bool is_target_detected


